terraform/helm/chaos/crd/README.md

```diff
@@ -0,0 +1,6 @@
+### Uploading Custom Resource Definitions (CRDs) to Kubernetes
+
+Example:
+```
+$ kubectl create -f crd-2.5.2.yaml
```

Comment:
https://helm.sh/docs/chart_best_practices/custom_resource_definitions/

Can we install it by default via helm?

Reply:
The [docs](https://chaos-mesh.org/docs/production-installation-using-helm/) say:
```
CAUTION
Currently, the latest CustomResourceDefinition (CRD) is not applied during the Helm upgrading, which might cause errors. To avoid this situation, you can apply the latest CRD manually:

curl -sSL https://mirrors.chaos-mesh.org/v2.5.2/crd.yaml | kubectl create -f -
```